test/nocks/email.routes.test.nock.js

```diff
@@ -262,89 +262,92 @@ export default function() {
         connection: 'close',
       },
     );
-}
 
-nock('https://so.api.mailgun.net:443')
```

Comment:
This was the source of my headache: this nock was outside the function, thus is was executed only one time when imported. The fact that it worked before was luck from tests execution order.